README.md

```diff
@@ -300,6 +301,34 @@ render((
 ), document.body);
 ```
 
+#### options.createElement - Custom React.createElement behavior
+
+Sometimes, you might want to override the `React.createElement` default behavior to hook into the rendering process before the JSX gets sent to `React.renderToString`. This might be useful to add extra children or modify some props based on runtime conditions. The function mirrors the `React.createElement` function, so the params are [`type, [props], [...children]`](https://reactjs.org/docs/react-api.html#createelement):
```

Comment:
"JSX gets sent to `React.renderToString`" -> "JSX is rendered"

Not everyone uses SSR, so I'd like the text to be a little more generic.